salt/_modules/metalk8s_volumes.py

```diff
@@ -85,7 +87,7 @@ def provision(name):
         name (str): volume name
 
     Returns:
-        str: path to the associated loop device
+        None
```

Comment:
Then I guess this `Returns: None` can be removed altogether?

Reply:
Done